packages/web3-eth/src/index.ts

```diff
@@ -0,0 +1,371 @@
+import { TransactionCall, TransactionWithSender, Web3BaseProvider } from 'web3-common';
+import { Web3Config, Web3RequestManager } from 'web3-core';
+import {
+	BlockNumberOrTag,
+	ValidTypes,
+	ValidReturnTypes,
+	convertToValidType,
+	Address,
+	Uint256,
+	HexString32Bytes,
+	HexStringBytes,
+	Uint,
+	HexString8Bytes,
+	convertObjectPropertiesToValidType,
+	isHexString32Bytes,
+	Filter,
+} from 'web3-utils';
+import {
+	convertibleBlockProperties,
+	convertibleFeeHistoryResultProperties,
+	convertibleReceiptInfoProperties,
+	convertibleTransactionInfoProperties,
+} from './convertible_properties';
+
+import * as RpcMethods from './rpc_methods';
+
+export class Web3Eth {
```

Comment:
Mentioned in PR description, but the `returnType` functionality is untested and the formatting of user input is not implemented for below methods